### PR TITLE
[no-ticket] [bugfix] Missing export on mgov logo caused build to fail 

### PR DESCRIPTION
**Pull request by forrestbaer**
## Summary

`yarn build:medicare` is currently failing in master because of a missing export on a props interface for `MedicaregovLogo`

## How to test

`yarn install && yarn build && yarn build:medicare`

### If this is a change to code:

- [x] Prefixed the PR title with the [Jira ticket number](https://jira.cms.gov/projects/WNMGDS/) as `[ticket] title`
- [x] Ran `yarn type-check && yarn lint && yarn test` with no errors
- [x] Updated unit test snapshots and loki reference images **if necessary** with `yarn loki` && `yarn update-snapshots`

### CODE REVIEW

**Review by scheul93 — APPROVED** [review]

**Comment by pwolfert** [comment]
Any idea where this came from? I see the error too, and I'm trying to figure out what changed

**Comment by pwolfert** [comment]
Is the stories module getting compiled by our build system and its doc generator running and failing to get that type?

**Comment by pwolfert** [comment]
It does look like our `.stories.` files are being included in the build, and I think we should probably ignore them.

**Comment by forrestbaer** [comment]
> It does look like our `.stories.` files are being included in the build, and I think we should probably ignore them.

Yeah, updating the tsconfig and testing.

File: packages/design-system-scripts/gulp/build.js [hunk 1]
```diff
@@ -23,6 +23,7 @@ const getSrcGlob = (src, changedPath) =>
     ? [changedPath]
     : [
         `${src}/**/*.{js,jsx,ts,tsx}`,
+        `!${src}/**/*.stories.{js,jsx,ts,tsx}`,
```
Comment (pwolfert) [inline packages/design-system-scripts/gulp/build.js:26]: When you don't fix the logo module, you also need to update line 114. You might as well also update 72 to match. I was modifying the existing ignores for test and definition files like ``!${src}/**/*{.test,.spec,.d,.stories}.{js,jsx,ts,tsx}`,`

**Review by pwolfert — APPROVED** [review]
One suggestion but otherwise LGTM

File: packages/design-system-scripts/gulp/build.js (hunk 1 above)
Comment (forrestbaer) [reply packages/design-system-scripts/gulp/build.js]: Ah, yes, good point, updated.

**Pull request merged**